js/google_ima.js

```diff
@@ -425,7 +425,8 @@ require("../html5-common/js/utils/utils.js");
           //displaying it, so just notify AMC that we are showing an overlay.
           else
           {
-            _amc.notifyNonlinearAdStarted(this.currentAMCAdPod.id);
+            // raise WILL_PLAY_NONLINEAR_AD event and notify nonlinear ad started
+            _amc.sendURLToLoadAndPlayNonLinearAd(this.currentAMCAdPod, this.currentAMCAdPod.id, this.currentAMCAdPod.ad.tag_url);
```

Comment:
In regards to the url, I was not aware of its implications. I saw that in
vast, the streamurl was passed into the sendURL function. Can you guys tell
me what the difference is?

And yes, sendURL does have the notify fun toon within it so the sendURL
does replace it.
On Dec 1, 2015 3:14 PM, "gfrankOoyala" <notifications@github.com> wrote:

> In js/google_ima.js
> <https://github.com/ooyala/html5-ad-plugins/pull/16#discussion_r46354703>:
>
> > @@ -425,7 +425,8 @@ require("../html5-common/js/utils/utils.js");
> >            //displaying it, so just notify AMC that we are showing an overlay.
> >            else
> >            {
> > -            _amc.notifyNonlinearAdStarted(this.currentAMCAdPod.id);
> > +            // raise WILL_PLAY_NONLINEAR_AD event and notify nonlinear ad started
> > +            _amc.sendURLToLoadAndPlayNonLinearAd(this.currentAMCAdPod, this.currentAMCAdPod.id, this.currentAMCAdPod.ad.tag_url);
>
> That is called by sendURL, I just double checked.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/html5-ad-plugins/pull/16/files#r46354703>.
>